packages/core/src/report/generate-report.ts

```diff
@@ -25,7 +17,35 @@ function getValues(dependencyTree: IArboristNode) {
   );
 }
 
-export async function generateReport(cwd: string): Promise<Report> {
+function processPlugins(plugins: string[]): PluginServer[] {
+  const processedPlugins: PluginServer[] = [];
+
+  plugins.forEach((pluginPath) => {
+    // Waiting for Lewis to tell me this is a bad idea
+    const { PluginServer } = require(pluginPath);
```

Comment:
I would use naming `ServerPlugin` instead since `PluginServer` reads as if you are creating a server for plugins

Reply:
makes sense, I can make this change.